frontend/src/container/AlertHistory/Timeline/Table/Table.tsx

```diff
@@ -1,31 +1,25 @@
 import './Table.styles.scss';
 
 import { Table } from 'antd';
-import { initialFilters } from 'constants/queryBuilder';
-import { REACT_QUERY_KEY } from 'constants/reactQueryKeys';
 import {
 	useGetAlertRuleDetailsTimelineTable,
 	useTimelineTable,
 } from 'pages/AlertDetails/hooks';
-import { useMemo, useState } from 'react';
+import { useMemo } from 'react';
 import { useTranslation } from 'react-i18next';
-import { useQueryClient } from 'react-query';
-import { PayloadProps } from 'types/api/alerts/get';
-import { TagFilter } from 'types/api/queryBuilder/queryBuilderData';
 
 import { timelineTableColumns } from './useTimelineTable';
 
 function TimelineTable(): JSX.Element {
-	const [filters, setFilters] = useState<TagFilter>(initialFilters);
-
 	const {
 		isLoading,
 		isRefetching,
 		isError,
 		data,
 		isValidRuleId,
 		ruleId,
-	} = useGetAlertRuleDetailsTimelineTable({ filters });
+	} = useGetAlertRuleDetailsTimelineTable();
+	// { filters }
```

Comment:
please make sure to remove all the unwanted comments once when the final PR against develop is ready

Reply:
Done